Add php and gcab to build dependency list

bobjrsenior opened the pull request:
When running "make everything", the last make step ("cab_file") requires both php and gcab. Without them, the image files still get made but the build is marked as a failure.

I added the php and gcab dependencies in the existing list but it seems that list came from the Yocto docs so I'm not sure if there should be a separate non-Yocto dependency list to make future changes easier.

I also had to change `pylint3` to just `pylint` but I don't know if that's just my system so I didn't change it in the list.

Biktorgj commented:
Those dependencies are only required when building packages for the LVFS, I should probably get all that out of the 'make everything' command as only people doing local updates with fwupd for testing purposes, or me getting a new release are going to make use of that.

`pylint3` could be changed to `pylint` as most distros should have an alias for it, but if you don't mind, I think it would be better to keep the dependencies for the SDK and fwupd specific stuff separate

bobjrsenior committed:
Add fwupd build dependencies, pylint3->pylint

When running "make everything", the last make step ("cab_file")
requires both php and gcab. Without them, the image files still get
made but the build is marked as a failure.

php and gcab are needed to make a build for fwupd in the last make step
so they were added to the build dependencies in a new step.

The pylint3 build dependency was updated to be pylint since most
distributions have pylint3 as an alias for pylint and some (at least
Ubuntu and Pop!_OS) don't have pylint3.

bobjrsenior commented:
I didn't expect the issue to close when I did that force-push. I'm pretty sure I've done that once without issue/PR changes.

Anyway, I was wondering what the cab file was for. Keeping the dependencies separate works for me.

I overwrote my commit (to keep history clean since it's a minor change) with one that changes `pylint3` to `pylint` and moves the fwupd specific dependencies (`php` and `gcab`) into a separate step.

Biktorgj commented:
Merged, thank you!
